export/distro/influxdb.go

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright (c) 2018
+// IOTech
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package distro
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"github.com/influxdata/influxdb/client/v2"
+)
+
+type influxdbSender struct {
+	client   client.Client
+	httpInfo client.HTTPConfig
+	database string
+}
+
+func NewInfluxDBSender(addr models.Addressable) Sender {
+	connStr := "http://" + addr.Address + ":" + strconv.Itoa(addr.Port)
+
+	influxdbHTTPInfo := client.HTTPConfig{
+		Addr:     connStr,
+		Timeout:  time.Duration(60000) * time.Millisecond,
```

Comment:
Do not hardcode numeric constants. No magic numbers, please declare a const.

Reply:
Will do.